spec/models/comment_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let!(:user) { create(:user) }
```

Comment:
I pretty sure that you don't need to create all this objects user, challenge and solution. ShouldaMatchers will do that for you.